improver/psychrometric_calculations/hail_size.py

```diff
@@ -559,15 +534,15 @@ def make_hail_cube(
 
         Args:
             hail_size
-                An n dimensional array of the diameter of hail stones (mm)
+                An n dimensional array of the diameter of hail stones (m)
             ccl_temperature
                 Cube of cloud condensation level pressure
             ccl_pressure
                 Cube of cloud condensation level pressure
             temperature_on_pressure
                 A cube of temperature on pressure levels
         Returns:
-            A cube of the diameter of hail stones(mm)
+            A cube of the diameter of hail stones(m)
```

Comment:
For my obsessive side...
```suggestion
            A cube of the diameter of hail stones (m)
```

Reply:
Done